server/modules/exchangeAccessCodeWithSlack.js

```diff
@@ -0,0 +1,35 @@
+const { WebClient } = require("@slack/web-api");
+const authenticationWithSlack = require("./authenticationWithSlack");
+
+const client = new WebClient();
+
+const slack_client_id = process.env.SLACK_CLIENT_ID;
+const slack_client_secret = process.env.SLACK_CLIENT_SECRET;
+
+const exchangeAccessCodeWithSlack = async function (req, res) {
+  const state = req.query.state;
+  // if (!state || !authenticationWithSlack.validate(state)) {
```

Comment:
Note: Team decided to move this to another ticket to fix the state validation - I think it'll be moving the createIdentifier function to somewhere both modules can access it.